action: broken
packages:
 - linux-64/pydantic-core-2.14.6-py310hcb5633a_0.conda
 - linux-64/pydantic-core-2.14.6-py311h46250e7_0.conda
 - linux-64/pydantic-core-2.14.6-py312h4b3b743_0.conda
 - linux-64/pydantic-core-2.14.6-py38h0cc4f7c_0.conda
 - linux-64/pydantic-core-2.14.6-py39h9fdd4d6_0.conda
 - linux-64/pydantic-core-2.14.6-py39he9b4010_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py310hfd680ee_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py311h2bdde80_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py312h3abe38b_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py38h4577bcc_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py39hf916c31_0.conda
 - linux-aarch64/pydantic-core-2.14.6-py39hfa7fa47_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py310hda86fbf_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py311hd3bf64d_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py312hbecd2b6_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py38h42aef6a_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py39h17c0542_0.conda
 - linux-ppc64le/pydantic-core-2.14.6-py39h819f8f7_0.conda
